server/src/main/java/org/opensearch/cluster/routing/allocation/Constraint.java

```diff
@@ -40,6 +40,10 @@ public void setEnable(boolean enable) {
         this.enable = enable;
     }
 
+    public boolean isEnable() {
```

Comment:
Can this be package private if only used in unit tests?

Reply:
done.